network/network.go

```diff
@@ -427,9 +448,18 @@ func (n *network) HealthCheck(context.Context) (interface{}, error) {
 	if !isMsgFailRate {
 		errorReasons = append(errorReasons, fmt.Sprintf("messages failure send rate %g > %g", sendFailRate, n.config.HealthConfig.MaxSendFailRate))
 	}
+
+	if !reachablePrimaryNetworkValidator {
+		errorReasons = append(errorReasons, "primary network validator is unreachable")
```

Comment:
I think this is the issue we are trying to catch, but I think the error should be more specific (to make potentially false positives easier to diagnose as false positives)
```suggestion
		errorReasons = append(errorReasons, "primary network validator has no inbound connections")
```

Reply:
I thought unreachable implies just that, that it's not reachable, but doesn't say anything about whether it can reach other nodes. 

Fixed [here](https://github.com/ava-labs/avalanchego/pull/3719/commits/9c9ca56e1ad115a5e5ba025a4df288e978e4ec12#diff-96a7f5c352cd3d2d1ee3c7c55ed95e668c6edb28a4fa337e6c60270471bdab61L453).

Reply:
The reason I prefer "has no inbound connections" is because it doesn't assume that no inbound connections implies unreachable. (For example, maybe the node is reachable.. But the public IP isn't set correctly. So the node could be reachable - just not at the broadcasted public IP).